pstatus-notifications-workflow-ktor/src/main/kotlin/Application.kt

```diff
@@ -6,24 +6,36 @@ import gov.cdc.ocio.processingnotifications.model.UploadDigestSubscription
 import gov.cdc.ocio.processingnotifications.service.UploadDigestCountsNotificationSubscriptionService
 import io.ktor.serialization.jackson.*
 import io.ktor.server.application.*
+import io.ktor.server.config.*
 import io.ktor.server.engine.*
 import io.ktor.server.netty.*
 import io.ktor.server.plugins.contentnegotiation.*
 import io.ktor.server.routing.*
 import org.koin.core.KoinApplication
+import org.koin.dsl.module
 import org.koin.ktor.plugin.Koin
 
 fun KoinApplication.loadKoinModules(environment: ApplicationEnvironment): KoinApplication {
     val databaseModule = DatabaseKoinCreator.moduleFromAppEnv(environment)
-    return modules(listOf(databaseModule))
+    val healthModule = module {
```

Comment:
Can we rename this to temporalConfigModule?

Reply:
Removed